diff --git a/cjs/react-dom-client.development.js b/cjs/react-dom-client.development.js
index 8c860aca7db6a1ca74788209d100f10ae5f085fa..6e6ee0a9ec6aeaa1f2791b23e13a9f3d2a349095 100644
--- a/cjs/react-dom-client.development.js
+++ b/cjs/react-dom-client.development.js
@@ -3965,8 +3965,13 @@
           (isDeeplyEqual = !1));
       for (var _key in next)
         if (_key in prev) {
-          var key = prev[_key];
-          var nextValue = next[_key];
+          var key, nextValue;
+          try {
+            key = prev[_key];
+            nextValue = next[_key];
+          } catch (e) {
+            continue;
+          }
           if (key !== nextValue) {
             if (0 === indent && "children" === _key)
               (isDeeplyEqual = "\u00a0\u00a0".repeat(indent) + _key),
